server/data/educationAndWorkPlanClient.ts

```diff
@@ -0,0 +1,24 @@
+import type { CreateGoalDto } from 'dto'
+import RestClient from './restClient'
+import config from '../config'
+import EducationAndWorkPlanApiMapper from './educationAndWorkPlanApiMapper'
+
+export default class EducationAndWorkPlanClient {
+  private static restClient(token: string): RestClient {
+    return new RestClient('Education and Work Plan API Client', config.apis.educationAndWorkPlan, token)
+  }
+
+  private static educationAndWorkPlanApiMapper(): EducationAndWorkPlanApiMapper {
+    return new EducationAndWorkPlanApiMapper()
+  }
+
+  async createGoal(createGoalDto: CreateGoalDto, token: string): Promise<void> {
+    const createGoalRequest =
```

Comment:
If you did as I suggested above re: the mapper, you can tidy this up. You can remove the static function altogether; and line 16 becomes:
```
const createGoalRequest = mapCreateGoalDtoToCreateGoalRequest(createGoalDto)
```